src/main/java/org/opensearch/agent/tools/CreateAnomalyDetectorTool.java

```diff
@@ -335,21 +336,8 @@ private Map<String, String> enrichParameters(Map<String, String> parameters) {
 
     /**
      *
-     * @param fieldsToType the flattened field-> field type mapping
-     * @return a list containing all the date type fields
-     */
-    private Set<String> findDateTypeFields(final Map<String, String> fieldsToType) {
-        Set<String> result = new HashSet<>();
-        for (Map.Entry<String, String> entry : fieldsToType.entrySet()) {
-            String value = entry.getValue();
-            if (value.equals("date") || value.equals("date_nanos")) {
-                result.add(entry.getKey());
-            }
-        }
-        return result;
-    }
-
     @SuppressWarnings("unchecked")
+    **/
```

Comment:
@SuppressWarnings should be outside of comment block. This may caused CI compile failure.